modules/arrow/src/arrow-writer.ts

```diff
@@ -18,7 +18,7 @@ export const ArrowWriter = {
   options: {}
 };
 
-function encodeSync(data, options = {}) {
-  const arrayBuffer = encodeArrowSync(data, options);
+function encodeSync(data) {
```

Comment:
I see, fine for now. I am not totally happy about this, as `options` is part of the type of the expected LoaderObject function.